docs/resources/compute_instance.md

```diff
@@ -262,6 +271,9 @@ The following arguments are supported:
   Defaults to *false*. This parameter is valid if `charging_mode` is set to *postPaid*, and all data disks will be deleted
   in *prePaid* charging mode.
 
+* `delete_eip_on_termination` - (Optional, Bool) Specifies whether the EIP is released when the instance is terminated.
```

Comment:
should eip associated with existing eip_id also be released? seems the codes below doesn't do this.

Reply:
as we discussed, the EIP will be released by the result of `shouldUnsubscribeEIP` function in pre-paid charging mode